### PR TITLE
refactor by using isXMMreg

### PR DESCRIPTION
**Pull request by kubo39**

### CODE REVIEW

File: src/dmd/backend/cod3.d [hunk 1]
```diff
@@ -3772,20 +3772,21 @@ void prolog_loadparams(ref CodeBuilder cdb, tym_t tyf, bool pushalloc, out regm_
                 if (!hasframe || (enforcealign && s.Sclass != SCshadowreg))
                     offset += EBPtoESP;
 
-                uint preg = s.Spreg;
+                // uint preg = s.Spreg;
```
Comment (jacob-carlborg) [inline src/dmd/backend/cod3.d]: Why a comment and not remove?

**Review by jacob-carlborg — APPROVED** [review]
LGTM, except for one minor detail.

File: src/dmd/backend/cod3.d (hunk 1 above)
Comment (kubo39) [reply src/dmd/backend/cod3.d]: Ahh, missed. Thank you, fix soon.

Comment (kubo39) [reply src/dmd/backend/cod3.d]: fixed

**Pull request merged**